### PR TITLE
WAZO-3423 search incalls by user_id

### PR DESCRIPTION
**Pull request by rbienvenu0**
Actual changes were done in the dao, this is only about documentation.

### CODE REVIEW

**Comment by rbienvenu0** [comment]
recheck

**Review by bloom1 — APPROVED** [review]
LGTM. Thank you!

File: integration_tests/suite/base/test_incalls.py [hunk 1]
```diff
@@ -165,6 +165,35 @@ def test_get(incall):
     )
 
 
+@fixtures.incall(id=0)
+@fixtures.user(id=1)
+@fixtures.incall(id=1)
+@fixtures.user(id=2)
+def test_get_userid(incall, user1, *_):
```
Comment (fblackburn1) [inline integration_tests/suite/base/test_incalls.py:172]: nit: you mix get / list nomenclature for tests. This test should be bellow `test_sorting_offset_limit` and renamed to `test_list_by_user_id`

File: integration_tests/suite/base/test_incalls.py [hunk 2]
```diff
@@ -165,6 +165,35 @@ def test_get(incall):
...
+                has_entries(
+                    {
+                        'moh_uuid': None,
+                        'user_id': user1['id'],
+                        'ring_time': None,
+                        'type': 'user',
+                        'user_firstname': user1['firstname'],
+                        'user_lastname': user1['lastname'],
+                    }
+                ),
```
Comment (fblackburn1) [inline integration_tests/suite/base/test_incalls.py:192]:
```suggestion
                has_entries(user_id=user1['id']),
```
This test should focus to only one thing, i.e. that result return the matches user_id

If you want to test if all arguments are there, use `test_get_user_destination_relation` (we can see there is some arguments missing in this test)

File: integration_tests/suite/base/test_incalls.py (hunk 1 above)
Comment (rbienvenu0) [reply integration_tests/suite/base/test_incalls.py]: Updated

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**